File: java/src/test/java/api/jdy/AppDemoTest.java
package api.jdy;

import api.jdy.AppDemo;

import java.util.Map;

/**
 * 应用 Demo 测试
 */
public class AppDemoTest {


    public static void main(String[] args) throws Exception {
        AppDemo appDemo = new AppDemo();
        // 应用列表 分页
        appList(appDemo);
        // 表单查询接口 分页
        entryList(appDemo);
    }

    private static void appList(AppDemo appDemo) throws Exception {
        Map<String, Object> result = appDemo.appList(0, 10);
        System.out.println("appList result \n" + result);
    }

    private static void entryList(AppDemo appDemo) throws Exception {
        Map<String, Object> result =  appDemo.entryList("6077edcf5820c900083c9b26", 0, 10);
        System.out.println("entryList result \n" + result);
    }
}

File: python/test/api/jdy/app_demo_test.py
import src.api.jdy.app_demo as app_demo
from src.constants.http_constant import HttpConstant


# 测试 应用分页列表
def appList():
    result = app_demo.appList(0, 10)
    print('appList result:', result)


# 表单查询接口 分页
def entryList():
    result = app_demo.entryList(HttpConstant.APP_ID, 0, 10)
    print('entryList result:', result)


if __name__ == '__main__':
    appList()
    entryList()
